ROADMAP.md

```diff
@@ -1,17 +1,7 @@
 Roadmap
 ---
-This represents the larger, bigger impact features and enhancements we have planned for K3s. Features are planned, but do not represent a commitment to develop and can change at any time. There are many more tactical enhancements and fixes that can be found by reviewing our [GitHub milestones](https://github.com/k3s-io/k3s/milestones).
+The k3s project uses [GitHub Milestones](http://github.com/k3s-io/k3s/milestones) to track the progress of changes going into the project.
 
-Next
----
-- Embedded registry support
-- Windows OS Support
-
-Later
----
-- Multus CNI support
-- Align K3s with upstream Kubernetes by removing patches or upstreaming them
-- FIPS-140-2 Compliant binaries
-- Graduate encrypted networking support from experimental to GA
-- Graduate network policy support from experimental to GA
+The k3s release cycle moves in cadence with upstream Kubernetes, with an aim to have new minor releases out within 30 days of upstream .0 releases.  To follow incoming changes, watching the [Backlog](https://github.com/orgs/k3s-io/projects/5) and [Current Development](https://github.com/orgs/k3s-io/projects/6) GitHub Projects is the most up to date way to see what's coming in upcooming releases.
 
+The development of k3s itself happens in the `main` branch, which correlates to the upcoming Kubernetes minor (currently 1.34).  These changes are then backported to the active release lines (at this time, `release-1.33`, `release-1.32`, and `release-1.31`)
```

Comment:
I wouldn't use the word currently here. In 6 months it will be out of date. Can this sentence be worded so it lasts for a long time?